### PR TITLE
Update (redo) zh_CN translation

### PR DESCRIPTION
**Pull request by Dobby233Liu**

### CODE REVIEW

**Comment by Rudloff** [comment]
Sorry for the delay. The fact that the order of some strings changed makes it a bit hard to see what exactly you added/changed.
I can see you replaced some translations. Can you please explain what you fixed?

**Comment by Dobby233Liu** [comment]
> Sorry for the delay. The fact that the order of some strings changed makes it a bit hard to see what exactly you added/changed. I can see you replaced some translations. Can you please explain what you fixed?

I think I translated some untranslated strings, and also redid some of them (some stuff are explained better now) and added more punctations.

File: i18n/zh_CN/LC_MESSAGES/Alltube.po [hunk 1]
```diff
@@ -1,134 +1,234 @@
...
+
+#: classes/Controller/FrontController.php:282
+msgid "Method not allowed"
+msgstr "不允许此请求方法"
+
+#msgid "yes"
+#msgstr "是"
 
+#msgid "no"
+#msgstr "否"
```
Comment (Rudloff) [inline i18n/zh_CN/LC_MESSAGES/Alltube.po:234]: Is this supposed to be commented?

Comment (Dobby233Liu) [reply i18n/zh_CN/LC_MESSAGES/Alltube.po:234]: Weird, i don't know why do I have that, i should probably remove them

**Pull request merged**